index.html

```diff
@@ -67,4 +69,5 @@ <h3>Browse Unit Accessibility Features!</h3>
     <footer class=" royal-blue">
         <a>&copy; 2024 KebabCase. All Rights Reserved.</a>
       </footer>
-</body>
\ No newline at end of file
+</body>
+</html>
```

Comment:
Remember, this red symbol means this file is missing a single blank line at the end of it.